### DISCUSSION TITLE
How can I get followers who blocked me?

### DISCUSSION THREAD

**Discussion by aslios** [sbertix/Swiftagram#149] [Q&A] [answered]
Hi Sbertix
i am using your api and i get my followers.But i want to find who blocked me in my followers.so how can i get followers who blocked me ?
Thank you for your help

**Comment by sbertix** [comment, marked as answer]
Hey @aslios, 

there's no way using the API itself… 
What you can do is cache your followers, then refresh the list and compare the two for changes. 

Of course any deletion might be due to a simple "unfollow", someone deactivating their account or they actually blocking you. 
You can narrow it down by checking for the existence of the account outside of the API (for instance inspecting and scraping their Instagram web page), then comparing it with your `Endpoint.User.summary` results. 
There's not a single way you can do this, and it mostly depends on your needs. 

I hope it's clear enough 😊

**Reply by aslios** [reply]
thank you so much :)
